src/labels/CustomLabels.labels

```diff
@@ -9519,6 +9519,19 @@ Once the process is 100% complete, you can safely leave or refresh this page.</v
         <shortDescription>stgHelpEnableSoftCreditRollups</shortDescription>
         <value>When selected, rolls up Closed/Won Opportunity totals to Contact records based on assigned Opportunity Contact Roles. Soft credit rollups only happen in nightly batches, but you can run them manually from the Rollup Donations Batch settings.</value>
     </labels>
+    <labels>
+        <fullName>stgHelpEnhancedRDBatch</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Enhanced Recurring Donation Batch Intro Help Text</shortDescription>
```

Comment:
![Screen Shot 2020-04-27 at 10 21 21 AM](https://user-images.githubusercontent.com/57271120/80402755-3d57c400-8873-11ea-9580-1ee2ae19efc7.png)
@sam-knox Please review the new custom label for the RD2 batch help text

Reply:
![Screen Shot 2020-04-27 at 11 21 38 AM](https://user-images.githubusercontent.com/57271120/80407110-f0c3b700-8879-11ea-80e7-ddaa8f45df8e.png)
@sam-knox Updated.